.builds/freebsd.yml

```diff
@@ -1,17 +1,26 @@
 image: freebsd/latest
+
 packages:
   - lang/python3
   - x11/libxcb
+
 sources:
   - https://github.com/alacritty/copypasta
+
+environment:
+  PATH: /home/build/.cargo/bin:/bin:/usr/bin:/usr/local/bin
```

Comment:
Seems like it broke build?

Reply:
Nah, tested without it too, doesn't work. It's the `test` that is broken the `build` works. I'll setup my FreeBSD VM tomorrow again when I get some time and see if I can get it working there without just hammering CI 24/7.